src/cuda-poh-verify/poh_verify.cu

```diff
@@ -0,0 +1,151 @@
+#include <stddef.h>
+#include <inttypes.h>
+#include <pthread.h>
+#include "gpu_common.h"
+#include "sha256.h"
+
+#define MAX_NUM_GPUS 8
+#define MAX_QUEUE_SIZE 8
+#define NUM_THREADS_PER_BLOCK 64
```

Comment:
Did you try different thread-block size?

Reply:
No, was planning to try different ones later with benchmarking.